.gitignore

```diff
@@ -207,17 +207,7 @@ $RECYCLE.BIN/
 *.out
 *.app
 
-
-### https://raw.github.com/github/gitignore/218a941be92679ce67d0484547e3e142b2f5f6f0/Global/VisualStudioCode.gitignore
-
-.vscode/*
-!.vscode/settings.json
-!.vscode/tasks.json
-!.vscode/launch.json
-!.vscode/extensions.json
-*.code-workspace
-
-# Local History for Visual Studio Code
-.history/
-
+inet/
+quisp/.vscode/
+quisp/modules/.vs/
```

Comment:
what is the `.vs` directory? do we need to exclude from git management?

Reply:
hmm, I've never seen this. we don't need it. could you remove it?

Reply:
yes